runtime/near-vm-runner/Cargo.toml

```diff
@@ -38,7 +38,7 @@ threadpool = "1.8.1"
 [dev-dependencies]
 near-test-contracts = { path = "../near-test-contracts" }
 assert_matches = "1.3"
-wat = "1.0"
+wat = "<=1.0.38"
```

Comment:
Pining to old wat is backwards, it's better to upgrade it instead. 

I think the difference boils down to wat generating (or not generating) a custom section with debug names for functions.

Reply:
> I think the difference boils down to wat generating (or not generating) a custom section with debug names for functions.

I believe you're correct, and that slight size changes compilation cost which is charged per byte. I'll pin it to latest wat, in case new version would change debug names again.

Reply:
I'd rather not pin -- pining creates an instant technical dept. If the change it again, we should raise an issue upstream asking about adding a mode which doesn't emit custom sections.